### PR TITLE
Updating lab guides after extenseive walkthoughs

### PR DESCRIPTION
**Pull request by nigelpoulton**
All lab guides walked through.

- **CICD Lab Guide**: Extensive changes made to the CICD guide. It was very complex and the pre-reqs required a lot of jumping in and out of other lab guides - I could see a lot of users getting lost and losing the will to live. Pre-req section entirely re-written so as to include all steps without having to refer to other lab guides. Further changes made throughout the doc to clarify things. It could do with another review some time in the future to give a more uniform look and feel but other fires are burning hotter right now.

- Other guides saw a lot of change as well to make them easier to follow and clearer in places where prior knowledge was assumed etc.

Signed-off-by: Nigel <nigelpoulton@hotmail.com>

### CODE REVIEW

**Comment by nigelpoulton** [comment]
@moxiegirl

File: 9-cicd-with-docker.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 > **Difficulty**: Advanced
 
 > **Time**: 75-90 minutes
-
+**NOTES:** This lab has a very complex setup - users have to go back to other labs and complete **parts** of them. This is quite disjointed and I imagine many lab users will get lost and frustrated.
```
Comment (nicolaka) [inline 9-cicd-with-docker.md]: @nigelpoulton yea i understand but can we remove this note. I think we should address the problem by making sure that the lab has a flow.

File: 9-cicd-with-docker.md [hunk 2]
```diff
@@ -27,107 +27,296 @@ In this lab, you will build and deploy [The Awesome Voting App](https://github.c
...
+        840c3c2529ca        docker/trusted-registry-distribution:v2.2.1    "registry /config/sto"   12 hours ago        Up 12 hours         5000/tcp                                   docker_trusted_registry_image_storage_0
+
+3. Browse to the URL of your **Node 0** over HTTPS
+
+	The URL of your **node-0** is the DNS name of your **node-0** AWS isntance and is provided to you as part of this lab. It will look something like `https://ec2-52-91-195-21.compute-1.amazonaws.com`
+
+4. Proceed insecurely when you get a certificate warning
+5. Configure **Domain name**, **Authentication**, and **License**
+    
+    Each of these operations is shown in steps 1 and 2 of Lab 3: Docker Orchestration.
```
Comment (nicolaka) [inline 9-cicd-with-docker.md]: should be Lab 3: DTR

**Comment by pvnovarese** [comment]
Changes to  5-content-trust.md LGTM

**Comment by anokun7** [comment]
LGTM

**Pull request merged**